qa/L0_model_update/instance_update_test.py

```diff
@@ -44,9 +45,21 @@ class TestInstanceUpdate(unittest.TestCase):
     __model_name = "model_init_del"
 
     def setUp(self):
-        # Initialize client
+        self.__reset_model()
         self.__triton = grpcclient.InferenceServerClient("localhost:8001")
 
+    def __reset_model(self):
```

Comment:
@Tabrizian is double underscore part of our code style?

Reply:
It is not. If you want to declare a private method, you can just use a single underscore. Double underscores are usually reserved for internal Python methods (e.g., `__init__`)

Reply:
I see, so we do not use https://docs.python.org/3.10/tutorial/classes.html#tut-private

Reply:
Updated: [Use _ as private](https://github.com/triton-inference-server/server/pull/5831/commits/10277054f99663f9f16293872953678d27d8fffb)